frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/cell/BranchOrCommitCellRendererComponent.java

```diff
@@ -313,14 +313,15 @@ private static String getSyncToParentStatusBasedToolTipText(INonRootManagedBranc
                 .format(currentBranchName, parentBranchName)),
```

Comment:
Nit: I'd make `sync-to-parent-status-tooltip.in-sync` string a HTML one as well for consistency (even if `<br>` isn't needed)